api/urls.py

```diff
@@ -20,6 +21,7 @@
 )
 
 internal_router = DefaultRouter()
+internal_router.register(r"groups", GroupViewSet, basename="groups")
```

Comment:
Are you sure it should be internal? As I see, it will be used by Dashboard. Then you probably should either edit PublicGroupViewSet somehow, or create a new one, but in `/public/` zone

Reply:
Because now "internal" means "not dashboard" (bot or blank DRF/curl interface)

Reply:
We definitely have to discuss permissions, and we even have TODOs about it :)

And also, I understand that our routers naming decision may be confusing, but this is what we decided

Reply:
And yes, it's fine to either do create route, or use default DRF create method (POST to /public/groups/) for this

Reply:
Added POST /public/groups for group creation, please, have a look one more time

Reply:
LGTM